manuscripts/report.py

```diff
@@ -666,6 +668,12 @@ def create_pdf(self):
         templates_path = os.path.join(os.path.dirname(__file__),
                                       "latex_template")
         copy_tree(templates_path, report_path)
+        # if user specified a logo then replace it with default logo
```

Comment:
I think using a default value for the logo is better than this check (see comments above). That way, it is more explicit which filename we're using by default.